README.md

```diff
@@ -8,7 +8,7 @@
 ## Features
 
 * Wait for GitHub Check Runs on a Pull Request
-* Configure the poll `interval` and `timeout`
+* Configure the poll `delay`, `interval` and `timeout`
```

Comment:
nit: Throughout this PR, can you put `delay` after the existing `interval` and `timeout` options? I don't expect most users to need it

Reply:
Done :)